docs/frontend.md

```
@@ -20,3 +20,13 @@ export const fetchOrganizations = async (): Promise<Array<Organization>> => {
   return validateResponse(response, isArray(isOrganization))
 }
 ```
+
+## Creating forms
+
+Forms are created using the useForm-hook from (react-hook-form)[https://react-hook-form.com/get-started]. Looking at already existing forms in the project is also a good starting point.
```

markdownlint-cli2 (0.17.2)

Reversed link syntax
(react-hook-form)